### PR TITLE
MCDU working menu page, atsu menu, basic weather request, basic aoc message functionality.

### PR DESCRIPTION
**Pull request by edwinbrowwn**
**Summary of Changes**

Updated menu page
Added active system status on menu page (green system highlights)
Added ATSU menu
Added AOC menu
Added weather request page
Weather request prefills departure and arrival airports
Weather request only supports live reports at the moment
Weather request only supports metars at the moment (taf, taf+metar coming next)
Weather request output displays in received messages
Basic message pagination implemented
Occasional system lag for realism

**Additional context**

Currently using aviationweather.gov for weather reports. Data requesting using reverse proxy.
This is a basic framework of the weather request system. All modifications are non breaking.
A few minor bugs may be noticed.

### CODE REVIEW

**Comment by tareksabet** [comment]
Can you provide us with screenshots of the result

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.html [hunk 1]
```diff
@@ -44,9 +44,20 @@
...
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_AocRequestsMessage.js"></script>
+ 
+<!-- MODIFIED BELOW 
+    <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_"></script> 
+=======
 
 <!-- MODIFIED BELOW
     <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_"></script>
+>>>>>>> master
 MODIFIED BELOW -->
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.html:61]: git conflict markers to remove here ;)

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.css [hunk 2]
```diff
@@ -138,7 +138,7 @@ fmc-main-display::before {
 
 .s-text {
   font-size: 4.8vw;
-  width: 90%;
+  width: 100%;
   letter-spacing: 0.9vw; }
```
Comment (Meqolo) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.css:142]: Closing bracket should be on the next line

**Comment by kiwi2021** [comment]
Please take a look at the conflict and we can merge this.

**Comment by edwinbrowwn** [comment]
Thanks I'm going to fix it today

**Comment by veikkos** [comment]
Great stuff, but please provide the screenshots if possible next time 👍

**Comment by AdenFlorian** [comment]
please provide before (if applicable) and after screenshots, and some IRL photo references that you used, thanks!

**Pull request closed without merging**